src/main/kotlin/no/skatteetaten/aurora/boober/feature/ToxiproxySidecarFeature.kt

```diff
@@ -24,22 +24,54 @@ import no.skatteetaten.aurora.boober.model.AuroraDeploymentSpec
 import no.skatteetaten.aurora.boober.model.AuroraResource
 import no.skatteetaten.aurora.boober.model.Paths.configPath
 import no.skatteetaten.aurora.boober.model.PortNumbers
+import no.skatteetaten.aurora.boober.service.CantusService
+import no.skatteetaten.aurora.boober.service.ImageMetadata
 import no.skatteetaten.aurora.boober.utils.addIfNotNull
 import no.skatteetaten.aurora.boober.utils.boolean
 
+private const val IMAGE_METADATA_CONTEXT_KEY = "imageMetadata"
```

Comment:
Kan context verdien bli overskrevet her? Brukes samme nøkkel for ToxiProxy, Job og fluent-bit.

Reply:
Eller med andre ord, har hver feature sin egen context?

Reply:
Hver feature har sin egen context. Dette har blitt flyttet inn i den abstrakte klassen.